vignettes/example_analysis.Rmd

```diff
@@ -1,104 +1,334 @@
 ---
-title: "Example analysis"
+title: "Example EQ-5D analysis using the surveyTools package"
 output: rmarkdown::html_vignette
 vignette: >
-  %\VignetteIndexEntry{Example analysis}
+  %\VignetteIndexEntry{Example EQ-5D analysis using the surveyTools package}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+bibliography: references.bib
 ---
 
 ```{r, include = FALSE}
+options(dplyr.summarise.inform = FALSE)
+
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>"
+  comment = "#>",
+  fig.align = "center",
+  fig.width = 8,
+  fig.height = 5
 )
+
+library(tidyverse)
```

Comment:
Will need to be explicit on packages used within vignette, e.g. dplyr, tidyr, surveyTools, etc ...